pkg/alpine/build-cache.sh

```diff
@@ -10,7 +10,7 @@ bail() {
 
 ALPINE_VERSION=$1
 
-if [ "$ALPINE_VERSION" != "edge" ]; then
+if [ "${ALPINE_VERSION#edge}" = "$ALPINE_VERSION" ]; then
```

Comment:
I am not sure I get this. First, why is this necessary? Second, does this fit with this PR? Should it not be on its own?

Reply:
I removed that and will submit a new PR for it.